clusterloader2/testing/experimental/storage/pod-startup/volume-types/persistentvolume/pvc.yaml

```diff
@@ -5,11 +5,15 @@ metadata:
   labels:
     app: {{.Name}}
     group: {{.Group}}
+{{ if .Provisioner }}
```

Comment:
Hmm - how does "ifs" for string work?

Reply:
Apparently it checks for non-empty, i.e. empty string = false.

Reply:
Interesting - but that's fine :)